sql/2021/mobile-web/color_contrast.sql

```diff
@@ -0,0 +1,12 @@
+#standardSQL
+# % mobile sites with sufficient text color contrast with its background
+SELECT
+  COUNTIF(color_contrast_score IS NOT NULL) AS total_applicable,
+  COUNTIF(CAST(color_contrast_score AS NUMERIC) = 1) AS total_sufficient,
+  COUNTIF(CAST(color_contrast_score AS NUMERIC) = 1) / COUNTIF(color_contrast_score IS NOT NULL) AS perc_in_applicable
```

Comment:
Is `perc_in_applicable` obvious what it is? Also we tend to use `pct` rather than `perc`

```suggestion
  COUNTIF(CAST(color_contrast_score AS NUMERIC) = 1) / COUNTIF(color_contrast_score IS NOT NULL) AS pct_sufficient_in_applicable
```